codemagic.yaml

```
@@ -0,0 +1,13 @@
+workflows:
+    production:
+        name: Deploy to production
+        environment:
+            groups:
+                - aws-production
+                - production
+        scripts:
+            - name: Deploy to production
+              script: |
+                  set -e
+                  aws s3 sync ./your-first-flutter-app-to-appstore s3://labs.codemagic.io --delete --cache-control max-age=31536000,public
```

I am not sure the intention was to publish this page as a main page
```suggestion
                  aws s3 sync . s3://labs.codemagic.io --delete --cache-control max-age=31536000,public
```

---

It is not a page, it is a [folder](https://github.com/codemagic-ci-cd/codelabs) with the index.html file